### PR TITLE
[TRANSLATION][SYSSETUP] Croatian translation and hotkey fix

### PR DESCRIPTION
**Pull request by Andrej123456789**
## Purpose
Translate 2nd stage of setup to Croatian and update `comctl` to fix hotkey

JIRA issue: N/A

## Proposed changes
Update few `.rc` files

_And yes, I watched on newlines._

## TODO
- [ ] Fix hotkey and translation on buttons (comctl) - I don't know why don't work

### CODE REVIEW

File: dll/win32/comctl32/lang/comctl_Hr.rc [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+LANGUAGE LANG_SERBIAN, SUBLANG_DEFAULT
```
Comment (HBelusca) [inline dll/win32/comctl32/lang/comctl_Hr.rc:21]: This needs to be fixed.

File: dll/win32/syssetup/syssetup.rc [hunk 2]
```diff
@@ -143,6 +149,9 @@ IDR_GPL RT_TEXT "COPYING"
 #ifdef LANGUAGE_ZH_CN
     #include "lang/zh-CN.rc"
 #endif
+#ifdef LANGUAGE_ZH_HK
```
Comment (HBelusca) [inline dll/win32/syssetup/syssetup.rc:152]: Why is there this change?

File: dll/win32/syssetup/syssetup.rc [hunk 3]
```diff
@@ -74,6 +74,9 @@ IDR_GPL RT_TEXT "COPYING"
 #ifdef LANGUAGE_ET_EE
     #include "lang/et-EE.rc"
 #endif
+#ifdef LANGUAGE_EU_ES
```
Comment (HBelusca) [inline dll/win32/syssetup/syssetup.rc:77]: Why is there this change?

File: dll/win32/comctl32/rsrc.rc [hunk 4]
```diff
@@ -117,6 +117,9 @@ IDI_TT_ERROR_SM ICON idi_tt_error_sm.ico
 #ifdef LANGUAGE_ET_EE
     #include "lang/comctl_Et.rc"
 #endif
+#ifdef LANGUAGE_EU_ES
```
Comment (HBelusca) [inline dll/win32/comctl32/rsrc.rc:120]: Why is there this change?

**Review by HBelusca — CHANGES_REQUESTED** [review]

File: dll/win32/comctl32/lang/comctl_Hr.rc (hunk 1 above)
Comment (Andrej123456789) [reply dll/win32/comctl32/lang/comctl_Hr.rc]: Uploaded code but still buttons don't work

**Comment by Andrej123456789** [comment]
I will open new Pull Request because my fork isn't updated for months.

**Pull request closed without merging**